hathor/p2p/sync_v2/mempool.py

```diff
@@ -39,6 +39,8 @@ def __init__(self, sync_agent: 'NodeBlockSync'):
         self.tx_storage = self.manager.tx_storage
         self.reactor = self.sync_agent.reactor
 
+        self.deferred: Optional[Deferred[None]] = None
```

Comment:
Should it be `self._deferred` since it's not meant to be used outside of the class?

Reply:
Done!